src/test/java/io/anserini/kg/freebase/FreebaseTest.java

```diff
@@ -0,0 +1,62 @@
+package io.anserini.kg.freebase;
+
+import org.junit.Test;
+
+import java.nio.file.Paths;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class FreebaseTest {
+  @Test(expected = NoSuchElementException.class)
+  public void test1() throws Exception {
+    Freebase freebase = new Freebase(Paths.get("src/test/resources/freebase-rdf-head100.gz"));
+
+    FreebaseNode node;
+    Iterator<FreebaseNode> iter = freebase.iterator();
+    assertTrue(iter.hasNext());
+    node = iter.next();
```

Comment:
But I believe than any test is better than no test...
You're welcome to contribute tests also... :)